packages/core/src/components/TextField/Styled/InnerWrapper/InnerWrapper.styled.tsx

```diff
@@ -40,6 +40,23 @@ const disabledStyle = ({ theme: { textField }, variant }: InnerWrapperProps) =>
     }
 `;
 
+const noDecoratorsStyle = () => css`
+    cursor: text;
+
+    ${CharacterCount} {
+        display: none;
+    }
+    ${Prefix}, ${Suffix} {
+        display: none;
+    }
+    & ~ ${HelperText} {
+        display: none;
+    }
```

Comment:
What do you think about moving this into the `TextField` component so that all of the logic for displaying these decorators is in the same place? For example:

```
 {isPrefixPresent && showDecorators && (
      <Styled.Prefix size={size}>
          <Prefix size={size} />
      </Styled.Prefix>
  )}
```